test/conformance/user_test.go

```diff
@@ -69,7 +70,11 @@ func TestMustRunAsUser(t *testing.T) {
 func TestShouldRunAsUserContainerDefault(t *testing.T) {
 	t.Parallel()
 	clients := setup(t)
-	_, ri, err := fetchRuntimeInfoUnprivileged(t, clients, &test.Options{})
+	_, ri, err := fetchRuntimeInfoUnprivileged(t, clients, &test.Options{
+		SecurityContext: &corev1.SecurityContext{
+			RunAsUser: ptr.Int64(1000),
```

Comment:
We specifically want to test falling back to the value specified in the image, so I don't think we want this.

Reply:
Yep, I realized that a second ago as well.